src/components/ApplicationInfo.tsx

```
@@ -0,0 +1,41 @@
+import React from "react";
+import { VStack } from "@chakra-ui/react";
+import { Table } from "ka-table"; // Importing ka-table directly
+import { DataType } from "ka-table/enums";
+interface ApplicationInfoProps {
+  details: { [key: string]: any };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)